src/main/java/core/basesyntax/file/writer/FileWriterImpl.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax.file.writer;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+public class FileWriterImpl implements FileWriter {
+    @Override
+    public void write(String data, String filePath) {
+        try {
+            Files.writeString(Paths.get(filePath), data);
+        } catch (IOException e) {
+            throw new RuntimeException("Cannot write to file: " + filePath, e);
+        }
+    }
```

Comment:
Follow the encapsulation principle. Consider making your class and its methods as restrictive as necessary. For example, if the class is not intended to be subclassed, declare it as final.